client/ayon_hiero/api/plugin.py

```diff
@@ -857,16 +856,24 @@ def _convert_to_tag_data(self):
 
             # adding tag metadata from ui
             for _key, _value in self.pre_create_data.items():
+                if (
+                    _key == "reviewableSource"
+                    and "reviewTrack" in self.tag_keys
+                ):
+                    self.tag_data.pop("reviewTrack")
+                    self.tag_data["reviewableSource"] = _value
                 if _key in self.tag_keys:
+                    # backward compatibility for reviewableSource (2024.11.08)
```

Comment:
shouldn't this comment be put to the new statement above ?

Reply:
moved upper, thanks